atef/widgets/config/run_base.py

```diff
@@ -251,32 +252,101 @@ def setup_buttons(self, configs, next_widget: TreeItem = None) -> None:
             self.setup_next_button(next_widget)
 
         self.setup_verify_button()
+        self.abort_button.clicked.connect(self.abort_task)
+        self.reveal_run_or_abort(running=False)
+
+    def reveal_run_or_abort(self, running: bool):
+        if running:
+            self.abort_button.show()
+            self.run_button.hide()
+        else:
+            self.abort_button.hide()
+            self.run_button.show()
 
     def _make_run_slot(self, configs) -> None:
 
         def run_slot(*args, **kwargs):
             """Slot that runs each step in the config list"""
             for cfg in configs:
+                self.run_button.hide()
+                self.abort_button.show()
```

Comment:
Suggestion: this should use `self.reveal_run_or_abort(running=True)` and should be outside the loop because it only needs to be done once.